src/components/VerschillenGebied.vue

```diff
@@ -285,23 +392,51 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@import "../../static/ams.scss";
+@import "~stijl/src/styles/generic/ams-colorpalette.scss";
 
-.map {
-  height: 350px;
-  margin-top: 5px;
-  margin-bottom: 5px;
-}
+  .map {
+    height: 400px;
+  }
 
-.select {
-  width: 100%;
-}
+  table {
+    /*override grid*/
+    width: 95% !important;
+  }
 
-.highlight-own {
-  color: $ams-blauw
-}
+  .highlight-own {
+    color: $ams-blauw;
+  }
+
+  .antwoord {
+    display: inline-block;
+    width: 25%;
+
+    input, label {
+      cursor: pointer;
+    }
+  }
+
+  .rij_verplicht {
+    +.alert-wrapper {
+      padding-top: 0;
+      margin-top: -1rem;
```

Comment:
Why do you use both em and px and not choose for either one of them?

Reply:
good one, i'll move to using rem everywhere (where possible)